libwallet/assets/wallet/wallet_shared.go

```diff
@@ -328,7 +328,11 @@ func (wallet *Wallet) SetBirthday(birthday time.Time) {
 func CreateNewWallet(pass *AuthInfo, loader loader.AssetLoader,
 	params *InitParams, assetType utils.AssetType,
 ) (*Wallet, error) {
-	seed, err := generateSeed(assetType)
+	if pass.WordSeedType.ToInt() == 0 {
+		return nil, errors.New("please select word seed type.")
```

Comment:
I guess my linter is complaining about the period here at the end of the error.
```suggestion
		return nil, errors.New("please select word seed type")
```